Commands/Search/GetSiteHistoricalVersionsStatus.cs

```diff
@@ -0,0 +1,56 @@
+﻿#if !ONPREMISES
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.SharePoint.Client;
+using Microsoft.SharePoint.Client.Search.Administration;
+using PnP.PowerShell.CmdletHelpAttributes;
+
+namespace PnP.PowerShell.Commands.Search
+{
+    [Cmdlet(VerbsCommon.Get, "PnPSiteHistoricalVersionsStatus", DefaultParameterSetName = "Xml")]
+    [CmdletHelp("Returns information about the Historical Versions feature for the current site collection from the context. " +
+       "The document statistics are only updated periodically, check the next update time property (in UTC) to see when new data will be available.",
+        SupportedPlatform = CmdletSupportedPlatform.Online,
+        Category = CmdletHelpCategory.Search)]
+    [CmdletExample(
+        Code = @"PS:> Get-PnPSiteHistoricalVersionsStatus",
+        Remarks = "Returns the status of the feature as well as the number of documents processed for the site if the feature is enabled.",
+        SortOrder = 1)]
+    public class GetSiteHistoricalVersionsStatus : PnPWebCmdlet
+    {
+
+        protected override void ExecuteCmdlet()
+        {
+            var siteLog = new SiteCrawlLog(ClientContext, ClientContext.Site);
```

Comment:
Are there any special permissions needed in order to use this object? If so we should add handling of this. @erwinvanhunen would it make sense to have this as a site extension method in Core, and use this here.. or just keep it for Posh?

Reply:
I set it up so that it uses essentially the same permissions checks as the Get-PnPSearchCrawlLog command. So I believe it could be fine to have it here.